demisto_sdk/tests/workflow_test.py

```diff
@@ -114,26 +121,28 @@ def run_validations(self):
         with ChangeCWD(self.content):
             runner = CliRunner(mix_stderr=False)
             self.run_command("git add .")
-            try:
-                # commit flow - secrets, lint and validate only on staged files without rn
-                res = runner.invoke(main, "secrets")
-                assert res.exit_code == 0
-                res = runner.invoke(main, "lint -g --no-test")
-                assert res.exit_code == 0
-                res = runner.invoke(main, "validate -g --staged --skip-pack-dependencies --skip-pack-release-notes "
-                                          "--no-docker-checks --debug-git")
-                assert res.exit_code == 0
-
-                # build flow - validate on all changed files
-                res = runner.invoke(main, "validate -g --skip-pack-dependencies --no-docker-checks --debug-git")
-                assert res.exit_code == 0
-
-                # local run - validation with untracked files
-                res = runner.invoke(main, "validate -g --skip-pack-dependencies --no-docker-checks --debug-git -iu")
-                assert res.exit_code == 0
-
-            except AssertionError:
-                raise AssertionError(f"stdout = {res.stdout}\nstderr = {res.stderr}")
+            # commit flow - secrets, lint and validate only on staged files without rn
+            res = runner.invoke(main, "secrets")
+            assert res.exit_code == 0, f"stdout = {res.stdout}\nstderr = {res.stderr}"
```

Comment:
we should add to each `assert` error message which command failed cause now it could be multiple commands.

This is right for all assertions in these tests

Reply:
pytest is already taking care of it.